app/src/androidTest/java/com/omen273/crossLingo/Utils.kt

```diff
@@ -94,10 +96,11 @@ fun generateCrossword(pressHome: Boolean = true, chooseTopics: List<Int> = listO
     onView(isRoot()).perform(waitForView(withId(R.id.ok_play)))
     onView(withId(R.id.ok_play)).perform(ViewActions.click())
     onView(isRoot()).perform(waitForView(withId(R.id.crossword)))
-    if (!pressHome) {
-        Espresso.pressBack()
-    } else {
+    when(finish) {
+        FinishTypeGenerateCrossword.PRESS_BACK -> Espresso.pressBack()
```

Comment:
what's the difference between back and home?

Reply:
Back is back and it is arrow on a right bottom angle home is home and it is arrow on a top left angle.